packages/manager/modules/pci/src/projects/new/classes/eligibility.class.js

```diff
@@ -33,4 +35,12 @@ export default class PciEligibility {
   isVerifyPaypalRequired() {
     return this.actionsRequired.includes(ELIGIBILITY_ACTION_ENUM.VERIFY_PAYPAL);
   }
+
+  setValidPaymentMethods(validPaymentMethods) {
+    this.validPaymentMethods = validPaymentMethods;
+  }
+
+  isDefaultPaymentMethodChoiceRequired() {
```

Comment:
I set it there in order to have all requirements at the same place. It simply extends a little bit the eligibility API response in order to add a new test.